src/components/BaseCard.vue

```diff
@@ -74,6 +74,9 @@ export default {
   },
 
   computed: {
+    isCreator() {
+      return get(this.post, 'relationships.creator.id', null).toString() === this.userState.id.toString();
```

Comment:
`let  creatorId = get(this.post, 'relationships.creator.id', '') 
return creatorId.toString() === this.userState.id.toString()`